instance/instance.go

```diff
@@ -125,6 +130,23 @@ func (s *Store) Update(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Get spatial link before overwriting links object
+	var spatial string
+	if instance.Links != nil {
+		if instance.Links.Spatial != nil {
+			if instance.Links.Spatial.HRef != "" {
+				spatial = instance.Links.Spatial.HRef
+			}
+		}
+	}
+
+	if spatial != "" {
+		instance.Links = currentInstance.Links
+		instance.Links.Spatial = &models.IDLink{
+			HRef: spatial,
+		}
+	}
```

Comment:
Feels a bit strange to do something specific just for the spatial link here. Should this logic live elsewhere? (this update function is getting quite long)